app/scripts/services/unit.coffee

```diff
@@ -347,6 +347,16 @@ angular.module('swarmApp').factory 'Unit', (util, $log, Effect, ProducerPaths, U
     #util.assert max.greaterThanOrEqualTo(0), "invalid unit cost max", @name
     return max
 
+  _costMetPercentOfVelocity: ->
+    max = new Decimal Infinity
+    for cost in @eachCost()
+      if cost.val.greaterThan(0)
+        console.log cost,cost.val.toNumber(),cost.unit.velocity().toNumber(),cost.unit.velocity().dividedBy cost.val
```

Comment:
Should use `$log.debug` or remove this, to avoid spamming production logs.

There's another three lines down.

lgtm otherwise, thanks!